program.txt

```diff
@@ -0,0 +1,14 @@
+### Text 1

```

Comment:
These you can actually add the citations to `README.md`. If you open that file you'll see that there are placeholders for  `### Text 1` and `### Text 2`